modules/oe_multilingual_url_suffix/src/Form/NegotiationUrlSuffixForm.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Drupal\oe_multilingual_url_suffix\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure the URL suffixes language negotiation method for this site.
+ *
+ * @package Drupal\oe_multilingual_url_suffix\Form
```

Comment:
We should remove all the `@package` notations, they don't make sense in a Drupal module.

Reply:
Done